src/main/java/com/epam/flightscontroltool/configuration/DataSourceConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.flightscontroltool.configuration;
+
+import com.zaxxer.hikari.HikariDataSource;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseDataSource;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import javax.sql.DataSource;
+
+@Configuration
+@Slf4j
+public class DataSourceConfig {
```

Comment:
https://docs.spring.io/spring-boot/docs/1.4.1.RELEASE/reference/html/howto-data-access.html
It is easier in this way